### ISSUE TITLE
[14351] Focused & Other tabs on portal tickets

### ISSUE THREAD

**Issue by Aycorn** [DeskDirector/UserVoice#264]
Feature request to make a service ticket “not visible” to a client.

For example, we don’t want our clients to see our recurring appointments within their portal ticket list.

If it is status “ Assigned-Onsite” AND “Type – Regular” we want to make that invisible to the customer.

**Comment by Nness** [comment]
I don't think this will be considered, unless someone have good reason for doing so. 

From user experiment point of view, when someone open an ticket, then at one stage, he cannot see that ticket inside portal anymore. What's the experience then?

If submit using email, he can always reply. No one can remove visibility of ticket from him. So why use portal?

**Comment by Nness** [comment]
I have thought through the scenario again, I'm still against hide ticket, but it make sense to have user importance level. Thus split open tickets to two collections might be good fit. Thus I have changed topic and descriptions accordingly. 

Use case scenario:
* MSP define set of status as non important, portal will not show them in focused collection but other.
* User can right click on ticket to mark it as unimportant or other, system automatically add `other` tag and it will move to other collection.

**Comment by Nness** [comment]
After discussing internally, I think there are scenarios that customer want to hide tickets from showing for end user. But status is not a good variable to control such behaviour. 

We feel ticket type or issue type is good fit for this. Still, by randomly switch type or issue type, could cause unexpected user experience. Personally, it is OK solution, but not perfect, there are many way can easily ruin the end user experience.

**Comment by iskolares** [comment]
Related request from another customer #25192 (since hiding by status was part of the initial request)

They are asking if we can hide tickets based on status. For example, any tickets set to "Closed - Spam" should not appear in the customer's ticket list.

They are also asking if we can add a "control" where they can select which ticket status the customer can see in their portal.
